Release And Bugfixing Strategy (#353)
secureCodeBox/secureCodeBox: SECURITY.md

# Security Policy

## Supported Versions

> Our _release cycle_ for new features (minior [semver](https://semver.org/lang/de/) update)

Reviewer: ```suggestion
Our _release cycle_ for new features (minior [semver](https://semver.org/) update)
```

is roughly every two weeks (we will usually make a new release after each review). 

| Version | Security Fixes* | Supported** |
| ------- | ------------------ | ------------------ |
| 2.6.x-alpha1 | :white_check_mark: | :white_check_mark: |
| 2.5.x | :white_check_mark: | :white_check_mark: |
| <= 2.4.x | :x: | :x: |
| < 2.0 | :x: | :x: |

### Major Release (Semver)
_Upcoming major updates_ will come with a time window in which both _major versions_ (starting with v2.x.x)
will receive security updates and bugfixes. The concrete support intervall will be probably a couple of months
and will be published when the next major version will be released.

### Minor Release/Feature Releases (Semver)

> We currently plan to provide support for the _latest minor [semver](https://semver.org/lang/de/)_ release only.

Reviewer: ```suggestion
We currently plan to provide support for the _latest minor [semver](https://semver.org/)_ release only.
```

### Patch Release/Bugfix/Security Fix
We try to make bugfixes and high severity fixes available as patch release for the current minor release
as early as possible.

## Extended (Enterprise) Support

> If you are interessted in extended support for older versions with security updates of our project 

Reviewer: ```suggestion
If you are interested in extended support for older versions with security updates of our project 
```

> please get in touch with the project team via Slack or eMail <secureCodeBox@iteratec.com>.

Reviewer: ```suggestion
please get in touch with the project team via Slack or email <secureCodeBox@iteratec.com>.
```

## Reporting a Vulnerability

> You have found a vulnerability in the project that shouldndt be discloused as public issue before its fixed?

Reviewer: ```suggestion
You have found a vulnerability in the project that shouldn't be disclosed as public issue before it's fixed?
```

> Please get in touch with the project team via Slack or eMail <secureCodeBox@iteratec.com>. 

Reviewer: ```suggestion
Please get in touch with the project team via Slack or email <secureCodeBox@iteratec.com>. 
```
